Add 2 missing commas

PedanticHacker opened the pull request:
Fixed the typing docs by adding 2 missing commas.

<!--
Thanks for your contribution!
Please read this comment in its entirety. It's quite important.

# Pull Request title

It should be in the following format:

```
bpo-NNNN: Summary of the changes made
```

Where: bpo-NNNN refers to the issue number in the https://bugs.python.org.

Most PRs will require an issue number. Trivial changes, like fixing a typo, do not need an issue.

# Backport Pull Request title

If this is a backport PR (PR made against branches other than `master`),
please ensure that the PR title is in the following format:

```
[X.Y] <title from the original PR> (GH-NNNN)
```

Where: [X.Y] is the branch name, e.g. [3.6].

GH-NNNN refers to the PR number from `master`.

-->

PedanticHacker committed:
Add 2 missing commas

Fixed the typing docs by adding 2 missing commas.

gvanrossum commented on Doc/library/typing.rst:
```diff
@@ -101,7 +101,7 @@ accidentally creating a ``UserId`` in an invalid way::
    output = UserId(23413) + UserId(54341)
 
-Note that these checks are enforced only by the static type checker. At runtime
+Note that these checks are enforced only by the static type checker. At runtime,
```
I think this sentence reads fine without the comma.

PedanticHacker replied:
This particular sentence needs a comma. But the second change where I added a comma was incorrect.

rhettinger reviewed (changes requested):
I think only the first of these two is correct.

PedanticHacker committed:
Fix my blunder

I've reverted my second change where I added a comma and realized that according to English orthographical rules it must not apply. The comma in the sentence "At runtime," is, however, necessary. Commas are not randomly placed by the writer according to "it feels right to me" but there are strict orthographical rules where each comma is placed in a sentence.
